Prefix/InfixToPrefixConverter.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Project2_Group_17.Prefix
+{
+    public class InfixToPrefixConverter
+    {
+        /// <summary>
+        /// Get the precedence of an operator
+        /// </summary>
+        /// <param name="op">The operator</param>
+        /// <returns>The precedence level</returns>
+        private static int Precedence(char op)
+        {
+            switch (op)
+            {
+                case '+':
+                case '-':
+                    return 1;
+
+                case '*':
+                case '/':
+                    return 2;
+
+                case '^':
+                    return 3;
+            }
+            return -1;
+        }
+
+        private static string infixToNearlyPostFix(string infix)
+        {
```

Comment:
What's nearly about it? Are the differences enough that we can't just use the Postfix conversion function we already have instead?

Also make sure to capitalize the whole function name (Infix...)

Reply:
I originally thought "wow easy I can just snake Hunter's post fix and it will work fine" but every answer would come out somewhat wrong once it got to complex operations. Basically the example he gave us does a "almost" postFix conversion but has a few caveats to handle prefix.
Also the function name is updated

Reply:
Damn, almost easy. Sounds good brother